contracts/ssnlist.scilla

```diff
@@ -409,7 +409,7 @@ end
 
 (* Check if the caller is the proxy *)
 procedure IsProxy()
-  proxy_tmp <- proxyaddr;
+  proxy_tmp = init_proxy_address;
```

Comment:
Why copy in the tmp variable? Now that it is not a mutable variable, you can directly call the builtin.

Reply:
Cool!

Reply:
Change pushed